### PR TITLE
Custom errors

### PR DESCRIPTION
**Pull request by sidneyjennings**
Added custom error pages for 404 and 500 errors. 500 error might need a redesign.

### CODE REVIEW

File: core/templates/core/500.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% extends 'core/base.html' %}
+{% block content %}
+{% load static %}
+<div class="container">
+    <div class="error404-message" style="font-size: 36px;">
```
Comment (jonahanderson) [inline core/templates/core/500.html:5]: This def doesnt matter but should this be error500-message

File: core/templates/core/500.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        UwU Oopsie Whoopsie. We did a fucky wucky, a fucko boingo.
+        <br>
+        Our engineers are working vewwwy hard to fix this. XD
```
Comment (jonahanderson) [inline core/templates/core/500.html:8]: everything looks good besides these lines but it kinda geeks me lol

**Comment by csyager** [comment]
Also, this has merge conflicts with master. On your local machine, do this:

```
git checkout master
git pull
git checkout custom_errors
git merge master
```

This will tell you what files have merge conflicts and mark them in the files. Go to those parts, keep the parts that should stay and which can be deleted (probably some code should stay from both), delete all the whacky >>>>>HEAD, ======, BRANCH<<<<<< stuff, then 
```
git add *
git commit -m “resolves merge conflicts”
```

And you should be good to go. I can help with this tomorrow if you can’t figure it out

File: core/templates/core/500.html (hunk 2 above)
Comment (csyager) [reply core/templates/core/500.html]: Just fixed this up.

**Pull request merged**